reference/appearance.md

```diff
@@ -23,10 +24,14 @@ lights are ignored during rendering of the object that references this
 [Appearance](#appearance)) and the unlit object color is (1,1,1).
 
 - The `texture` field, if specified, shall contain an
-[ImageTexture](imagetexture.md) node. If the `texture` node is NULL or the
+[ImageTexture](imagetexture.md) node or a [MultiTexture](multitexture.md) node.
+If the `texture` node is NULL or the
 `texture` field is unspecified, the object that references this
```

Comment:
This declaration is a copy paste from the X3D specification:
http://www.web3d.org/documents/specifications/19775-1/V3.3/Part01/components/shape.html#Appearance
But this is indeed not very clear.